### PR TITLE
ldapdomaindump: add page

### PR DESCRIPTION
**Pull request by dhondta**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/ldapdomaindump.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ldapdomaindump
+
+> Dumps users/computers/groups and OS/membership information via LDAP to HTML/JSON/greppable output.
+> More information: <https://github.com/dirkjanm/ldapdomaindump>.
+
+- Dump all information using the given LDAP account:
+
+`ldapdomaindump --user {{domain\\administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
```
Comment (blueskyson) [inline pages/linux/ldapdomaindump.md:8]:
```suggestion
`ldapdomaindump --user {{domain}}\\{{administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
```

I wonder that `--user` option accepts only administrators?

File: pages/linux/ldapdomaindump.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# ldapdomaindump
+
+> Dumps users/computers/groups and OS/membership information via LDAP to HTML/JSON/greppable output.
```
Comment (blueskyson) [inline pages/linux/ldapdomaindump.md:3]:
```suggestion
> Dumps information in formats of `users/computers/groups` and `OS/membership` via LDAP to HTML, JSON and greppable output.
```

File: pages/linux/ldapdomaindump.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`ldapdomaindump --resolve --user {{domain\\administrator}} --password {{password}} {{hostname|ip}}`
+
+- Dump all information, resolving computer hostnames with the selected DNS server:
+
+`ldapdomaindump --resolve --dns-server {{domain_controller_ip}} --user {{domain\\administrator}} --password {{password}} {{hostname|ip}}`
+
+- Dump all information to the given directory without JSON output:
+
+`ldapdomaindump --no-json --outdir {{output/path}} --user {{domain\\administrator}} --password {{password}} {{hostname|ip}}`
```
Comment (blueskyson) [inline pages/linux/ldapdomaindump.md:20]:
```suggestion
`ldapdomaindump --no-json --outdir {{path/to/directory}} --user {{domain}}\\{{administrator}} --password {{password}} {{hostname|ip}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for your contribution. Here are some suggestions.

File: pages/linux/ldapdomaindump.md (hunk 1 above)
Comment (dhondta) [reply pages/linux/ldapdomaindump.md:8]: You're completely right, thank you for the head up.

File: pages/linux/ldapdomaindump.md (hunk 2 above)
Comment (dhondta) [reply pages/linux/ldapdomaindump.md:3]: These are not formats !
I guess I could improve my sentence the following way : 
`Dumps users, computers, groups, OS and membership information via LDAP to HTML, JSON and greppable output.`

Comment (blueskyson) [reply pages/linux/ldapdomaindump.md:3]: Sure, go for it.

File: pages/linux/ldapdomaindump.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Dumps users, computers, groups, OS and membership information via LDAP to HTML, JSON and greppable output.
+> More information: <https://github.com/dirkjanm/ldapdomaindump>.
+
+- Dump all information using the given LDAP account:
+
+`ldapdomaindump --user {{domain}}\\{{administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
+
+- Dump all information, resolving computer hostnames:
+
+`ldapdomaindump --resolve --user {{domain\\administrator}} --password {{password}} {{hostname|ip}}`
```
Comment (blueskyson) [inline pages/linux/ldapdomaindump.md:12]:
```suggestion
`ldapdomaindump --resolve --user {{domain}}\\{{administrator}} --password {{password}} {{hostname|ip}}`
```

File: pages/linux/ldapdomaindump.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Dump all information using the given LDAP account:
+
+`ldapdomaindump --user {{domain}}\\{{administrator}} --password {{password|NTLM_hash}} {{hostname|ip}}`
+
+- Dump all information, resolving computer hostnames:
+
+`ldapdomaindump --resolve --user {{domain}}\\{{administrator}} --password {{password}} {{hostname|ip}}`
+
+- Dump all information, resolving computer hostnames with the selected DNS server:
```
[the remark this answers is not in the document]
Comment (dhondta) [reply pages/linux/ldapdomaindump.md:15]: Info dump is not made *with the selected DNS server*, but *resolving computer hostnames* is...
This sentence shall not be rephrased.

**Review by marchersimon — APPROVED** [review]
Thanks @dhondta

**Pull request merged**